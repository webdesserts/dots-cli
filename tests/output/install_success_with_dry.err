[info] Looks Good! Nothing wrong with the current install plan!
