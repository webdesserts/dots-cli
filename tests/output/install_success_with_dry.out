
[example_dot]
✔ shell/bashrc => ~/.bashrc
✔ shell/zshrc => ~/.zshrc

